chrome/ansible-navigation.json

```diff
@@ -154,36 +87,84 @@
+            "isExternal": true,
+            "filterable": false,
+            "href": "https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/"
+        },
+        {
+            "groupId": "operations",
             "icon": "trend-up",
-            "title": "Business Insights",
+            "title": "Red Hat Insights",
             "navItems": [
                 {
-                    "appId": "registration",
-                    "title": "Register Systems",
-                    "filterable": false,
-                    "href": "/ansible/registration",
+                    "appId": "inventory",
+                    "title": "Inventory",
+                    "href": "/ansible/inventory",
                     "product": "Red Hat Insights"
                 },
                 {
-                    "appId": "remediations",
-                    "title": "Remediations",
-                    "href": "/ansible/remediations",
-                    "product": "Red Hat Insights"
+                    "title": "Advisor",
+                    "expandable": true,
+                    "routes": [
+                        {
+                            "appId": "advisor",
+                            "title": "Recommendations",
+                            "href": "/ansible/advisor/recommendations",
+                            "product": "Red Hat Insights"
+                        },
+                        {
+                            "appId": "advisor",
+                            "title": "Systems",
+                            "href": "/ansible/advisor/systems",
+                            "product": "Red Hat Insights"
+                        },
+                        {
+                            "appId": "approval",
+                            "title": "Topics",
+                            "href": "/ansible/advisor/topics",
+                            "product": "Red Hat Insights"
+                        }
+                    ]
                 },
                 {
-                    "appId": "tasks",
-                    "title": "Tasks",
-                    "href": "/ansible/tasks",
-                    "product": "Red Hat Insights"
+                    "title": "Drift",
+                    "expandable": true,
+                    "routes": [
+                        {
+                            "appId": "drift",
+                            "title": "Comparison",
+                            "href": "/ansible/drift/",
+                            "product": "Red Hat Insights"
+                        },
+                        {
+                            "appId": "drift",
+                            "title": "Baselines",
+                            "href": "/ansible/drift/baselines",
+                            "product": "Red Hat Insights"
+                        }
+                    ]
+                },
+                {
+                    "appId": "policies",
```

Comment:
This item is not on the preview

Reply:
https://issues.redhat.com/browse/RHIF-38 - it's been discussed and Policies will be put back in, this change is correct.